### PR TITLE
jQuery: July 2023 Update

### PR DESCRIPTION
**Pull request by timmywil**

### CODE REVIEW

**Review by bensternthal — APPROVED** [review]

**Comment by timmywil** [comment]
@bensternthal I went ahead and created a README for 2023 updates that I'd like to include in the submission to ossf. I see you already reviewed, but I wanted to make sure you saw that.

**Review by bensternthal — APPROVED** [review]

**Comment by bensternthal** [comment]
@timmywil yep I saw that and thanks for creating!

File: alpha/engagements/2023/jquery/update-2023-07.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+We're currently investigating reports that some users have been unable to access the jQuery CDN via Tor. However, the CDN will soon be migrated from StackPath to Fastly, which may end up resolving the issue.
+
+## Decommissioning wp-01.stage
+
+This droplet was 8 years old and was no longer used. We've transitioned all pointers to this droplet to the new infrastructure and have turned it off after 1 week.
+
+## Profiling Traffic for Fastly
+
+In preparation to migrate the CDN to Fastly from StackPath and at Fastly's request, we're putting together a doc that profiles jQuery's current CDN traffic. This will help Fastly determine the best configuration for our CDN. The doc is currently in progress.
```
Comment (Krinkle) [inline alpha/engagements/2023/jquery/update-2023-07.md]: I suggest linking some words here to https://github.com/jquery/infrastructure-puppet/blob/staging/doc/cdn.md. These figures don't need to be private!

**Review by Krinkle — APPROVED** [review]

**Comment by timmywil** [comment]
Merged in https://github.com/jquery/alpha-omega/pull/2/commits/0b0f32fbd4ccd197b3eb5ec4e1ab8cc7f57cda57

**Pull request closed without merging**